cli/bin.ts

```diff
@@ -0,0 +1,23 @@
+import { download } from "../deps/capi_binary_builds.ts"
+
+export default async function(binary: string, version: string, ...args: string[]) {
+  if (!binary || !version) throw new Error("Must specify binary and version")
+
+  const binaryPath = await download(binary, version)
+
+  const child = new Deno.Command(binaryPath, {
+    args,
+    stdin: "inherit",
```

Comment:
this can also be omitted since inherit is the default

Reply:
The default varies by invocation, so it's clearest to have it explicit IMO